chart/Chart.tpl.yaml

```
@@ -4,6 +4,6 @@ description: Sample Chart Description
 
 type: application
 
-version: {{.Env.VERSION}}
+version: { { .Env.VERSION } }
```

this is wrong change
```suggestion
version: {{ .Env.VERSION }}
```

---

fixed